src/main/java/com/force/i18n/commons/util/collection/MapSerializer.java

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
See changes in [LanguageDictionary](https://github.com/salesforce/grammaticus/pull/30/files#diff-91aec7f03f9d533b88cf12eeac4f2771e64c2080b9a1992481bf30c17b8139a5)